templates/eap-cd-starter-s2i.json

```diff
@@ -0,0 +1,342 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "annotations": {
+            "iconClass": "icon-eap",
+            "tags": "eap,javaee,java,jboss",
+            "version": "1.5.0",
+            "openshift.io/display-name": "JBoss EAP CD Starter",
```

Comment:
JBoss EAP CD Starter Template?
JBoss EAP CD for OpenShift Online Starter?

Reply:
This seems fine as is. 

We don't add "Template" elsewhere so if we decided we wanted to add that we should do it everywhere as separate work. But I don't know if it adds much. In the OSO Starter catalog I don't see that word.

Re: adding 'OpenShift Online', it's a valid 'Starter' elsewhere, even if we don't plan to install it.